libs/components/flyout/src/lib/modules/flyout/flyout.component.scss

```diff
@@ -150,6 +151,7 @@ $header-height: 50px;
 @include mixins.sky-theme-modern-dark {
   .sky-flyout {
     background: #000;
+    --sky-background-color-page-default: #000;
```

Comment:
why do flyout and modal need to set the background color variable now?

Reply:
The designer wants `transparent` background, but when viewkeeper pins the header, `transparent` makes the scrolled content viewable under the header. I'm relying on `--sky-background-color-page-default` and updating it in contexts where the background changes, which is flyout and modal.